### PR TITLE
Feature update project name

### PR DESCRIPTION
**Pull request by franc-hoon**
Change project name to Mello (can always change later if anyone has better ideas. Mello cuz it's like Trello)
Add icon
mello

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Mello</title>
```
Comment (YungiKim1999) [inline public/index.html:27]: I was thinking HIIT Board (High Intensity Interval Tasking Board)

**Review by YungiKim1999 — APPROVED** [review]

**Review by thakurtushar02 — COMMENTED** [review]
It's OK, but the commit history is a bit screwed up. Could you create a new branch stemming from updated main and then do your last commit on it?

Otherwise it's good 👍

**Review by lucas2005gao — APPROVED** [review]
I love this logo

**Comment by lucas2005gao** [comment]
Hey @M-Stable why did we close this PR :P

**Pull request closed without merging**